lib/index.js

```diff
@@ -96,6 +103,21 @@ const updateAvailFiats = fiats => ({
   payload: fiats
 });
 
+const setFeed = feed => ({
+  type: 'SET_FEED',
+  payload: feed
+});
+
+const updateFiat = fiat => ({
+  type: 'UPDATE_FIAT',
+  payload: fiat
+});
+
+const refreshPrice = {
+  type: 'REFRESH_PRICE',
+  payload: {}
```

Comment:
payload isn't necessary if not passing anything. the `type` and `payload` properties are really just a redux convention. There's nothing that actually enforces this except the consumers of the action in the reducer itself. So if there's no payload, the consumer won't be checking for it anyway. These can technically be whatever you want though. bsock-middleware for example has a `bsock` property, which indicates that it has properties for a bsock socket. `thunk-middleware` checks for a returned function instead of an object. So it's kind of arbitrary :)